components/InfoBadge.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC, ReactNode } from 'react'
+import { Badge, BadgeProps } from '@chakra-ui/react'
+import { NAMED_COLORS } from 'theme/constants'
+
+interface InfoBadgeProps extends BadgeProps {
+  message: ReactNode
+}
+
+const InfoBadge: FC<InfoBadgeProps> = ({ message, ...rest }) => (
+  <Badge
+    bg={NAMED_COLORS.LIGHT_YELLOW}
+    color={NAMED_COLORS.BLACK}
+    borderRadius="5rem"
+    py="0.375rem"
+    px="1.125rem"
+    textTransform="none"
+    {...rest}
+  >
+    <h5>{message}</h5>
```

Comment:
issue: Sorry to be so nitpicky about this, but generic component should be context-agnositc. I.e. they should work regardless of where they're placed. If we agree that's true, hardcoding a heading level doesn't really make sense, as headings are supposed to go in order.

Also, since `<Badge>` is generic and takes an `as` prop, this means we could write something like

```
<InfoBadge as="button" message="Hello" />
```

which would also be invalid since buttons should not contain headings.

IMO, the preferred solution would be to simply omit the `<h5>` altogether. If you need the `<InfoBadge>` to render an `<h5>` you could simply do

```
<InfoBadge as="h5" message="Hello" />
```

Reply:
InfoBadge has its own font styles that overrides props of `as` component (at least for `h5`). Then seems there is no point in adding separate component.